src/components/view/news/news.ts

```diff
@@ -0,0 +1,63 @@
+import './news.css';
+
+export interface Article {
+    source: {
+        id: number,
+        name:string,
+    };
+    author: string;
+    title: string;
+    description: string;
+    url: URL;
+    urlToImage: URL;
+    publishedAt: string;
+    content: string;
+}
+
+class News {
+    draw(data = []) {
+        const news = data.length >= 10 ? data.filter((_item, idx: number) => idx < 10) : data;
+
+        const fragment = document.createDocumentFragment();
+        const newsItemTemp = document.querySelector('#newsItemTemp') as HTMLTemplateElement;
+        const newsDiv = document.querySelector('.news') as Element;
+
+        news.forEach((item: Article, idx: number) => {
+            const newsClone = newsItemTemp.content.cloneNode(true) as Element;
```

Comment:
```suggestion
            const newsClone: HTMLElement | null = newsItemTemp.content.cloneNode(true);
```
Here and bellow

Reply:
rewritten as:
const newsClone = <HTMLDivElement>newsItemTemp.content.cloneNode(true);